server/src/main/java/io/littlehorse/common/model/getable/global/acl/TenantModel.java

```diff
@@ -0,0 +1,74 @@
+package io.littlehorse.common.model.getable.global.acl;
+
+import com.google.protobuf.Message;
+import io.littlehorse.common.model.AbstractGetable;
+import io.littlehorse.common.model.GlobalGetable;
+import io.littlehorse.common.model.getable.objectId.TenantIdModel;
+import io.littlehorse.common.proto.TagStorageType;
+import io.littlehorse.common.proto.Tenant;
+import io.littlehorse.sdk.common.exception.LHSerdeError;
+import io.littlehorse.server.streams.store.ModelStore;
+import io.littlehorse.server.streams.storeinternals.GetableIndex;
+import io.littlehorse.server.streams.storeinternals.index.IndexedField;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Getter
+@EqualsAndHashCode(of = "id", callSuper = false)
+public class TenantModel extends GlobalGetable<Tenant> {
+
+    private String id;
+
+    public TenantModel() {}
+
+    public TenantModel(final String id) {
+        this.id = id;
+    }
+
+    public static TenantModel createDefault() {
```

Comment:
This belongs in the `DAO`, not in the `TenantModel`. That will enable the DAO to determine whether the `DefaultTenant` has been deleted.

And again, creating stuff willy-nilly in memory is not a good idea especially when we add more fields to the `Tenant`.